src/linuxcan.cpp

```diff
@@ -278,17 +279,12 @@ std::vector<std::shared_ptr<KvaserCard>> KvaserCanUtils::getCards()
 
 std::vector<std::shared_ptr<KvaserChannel>> KvaserCanUtils::getChannels()
 {
-  std::vector<std::shared_ptr<KvaserChannel>> channels;
-
   int32_t numChan = -1;
-  ReturnStatuses retStat = ReturnStatuses::OK;
-
-  retStat = KvaserCanUtils::getChannelCount(&numChan);
+  std::vector<std::shared_ptr<KvaserChannel>> channels;
+  KvaserCanUtils::getChannelCount(&numChan);
 
   // Sanity checks before continuing
-  if (retStat == ReturnStatuses::OK &&
-    numChan > -1 &&
-    numChan < 300)
+  if (numChan > -1 && numChan < 300)
```

Comment:
Where does 300 come from?

Reply:
From Kvaser's examples. I guess they made the assumption that a system could never have more than 300 devices on it (though the number is still arbitrary). Figured I would just follow their example here.